lib/spoom/sorbet.rb

```diff
@@ -34,7 +55,14 @@ def srb(*arg, path: ".", capture_err: false, sorbet_bin: nil)
         else
           arg.prepend("bundle", "exec", "srb")
         end
-        Spoom.exec(*T.unsafe(arg), path: path, capture_err: capture_err)
+        result = Spoom.exec(*T.unsafe(arg), path: path, capture_err: capture_err)
+
+        case result.exit_code
+        when SEGFAULT_CODE
+          raise Error::Segfault.new("Sorbet segfaulted.", result)
+        end
```

Comment:
NIT: do we need a case if we only have a single when? How about switching to an if statement?